src/main/xerus/monstercat/MonsterUtilities.kt

```diff
@@ -99,6 +95,7 @@ class MonsterUtilities(checkForUpdate: Boolean): JFXMessageDisplay {
 			}
 		}
 		addTab(TabCatalog::class)
+		addTab(TabPlaylist::class)
```

Comment:
Not quite sure that that's the right position. I think it belongs right before Settings.

Reply:
It's next to catalog, which is the place from which you add songs to it. Makes sense to me ?

Reply:
What if you add something from the Downloader? And either way, it's the weirdo on the block, because the other tabs mind their own business but Playlist is only there for the Player.

Reply:
I'm open to other ideas.

Reply:
My opinion stays, right before Settings. Until you add it to the advanced player controls ;)

Reply:
82b92797a9cf5a05a54aab7a97c067fcdb794ccb